Multisig/UI/Transaction/ActionDetailAddressCell.swift

```diff
@@ -11,6 +11,6 @@ import UIKit
 class ActionDetailAddressCell: ActionDetailTableViewCell {
     @IBOutlet private weak var addressView: AddressInfoView!
     func setAddress(_ value: Address) {
-        addressView.setAddress(value, label: nil)
+        addressView.setAddress(value, label: nil, imageUri: nil)
```

Comment:
agree with Mouaz, default value = nil (also for label) will make it nicer

Reply:
done